### PR TITLE
Objects Automated Test - LPS-152058, LPS-152059, LPS-152060, LPS-152064 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-152058
https://issues.liferay.com/browse/LPS-152059
https://issues.liferay.com/browse/LPS-152060
https://issues.liferay.com/browse/LPS-152064

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

**Review by timpak — CHANGES_REQUESTED** [review]
Hey @andreefarias, please see my comments below. You can send me another pull request once you make the necessary changes. Let me know if you need help or if something doesn't make sense!

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 1]
```diff
@@ -72,12 +72,81 @@ definition {
...
+		for (var resourceAction : list "UPDATE,VIEW") {
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "${resourceAction}",
+				resourceName = "com.liferay.object.model.ObjectDefinition",
+				roleTitle = "Regular Role");
+		}
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
```
[the remark this answers is not in the document]
Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase:117]: Hi, @timpak!

The role Power User don't check Objects proprieties by default. So, in some tests is necessary to mark the properties that will be use. If not, the user will not be able to make any changes relate Objects or Custom Objects.

[the remark this answers is not in the document]
Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase:117]: But, is possible change the creation Regular Role to Power User role.

**Comment by timpak** [comment]
See https://github.com/timpak/liferay-portal/pull/225

See my changes https://github.com/timpak/liferay-portal/pull/225/commits/395e605b2d457fde54310c1fae1551bf9f810f45 and https://github.com/timpak/liferay-portal/pull/225/commits/0bd60f969309f98fdb42972a829409104e40b0b5

**Pull request closed without merging**